src/schema/common.ts

```diff
@@ -148,6 +149,25 @@ export const getSearchQuery = (param: string): string =>
 export const processSearchQuery = (query: string): string =>
   query.trim().split(' ').join(' & ') + ':*';
 
+export const meiliOffsetGenerator = <TReturn>(): PageGenerator<
+  TReturn,
+  unknown,
+  OffsetPage,
+  unknown
+> => ({
+  connArgsToPage: ({ pagination }): OffsetPage => {
+    return {
+      limit: pagination?.total || 10,
+      offset: pagination?.offset + 1 || 0,
+      current: pagination?.current,
+    };
+  },
+  nodeToCursor: (page, args, node, i): string =>
+    offsetToCursor(page.offset + i),
+  hasNextPage: (page): boolean => page.current <= page.limit,
```

Comment:
next page is if total is lower then current right?

Reply:
eg. `estimatedTotalHits`

Reply:
yeah limit is `estimatedTotalHits` and current = current fetched total.
Hope that makes sense?